otter/convergence/steps.py

```diff
@@ -222,4 +228,56 @@ def as_effect(self):
         Produce a :obj:`Effect` to remove some nodes from some RCv3 load
         balancers.
         """
-        return _rackconnect_bulk_request(self.lb_node_pairs, "DELETE", (204,))
+        eff = _rackconnect_bulk_request(self.lb_node_pairs, "DELETE",
+                                        success_pred=has_code(204, 409))
+        # While 409 isn't success, that has to be introspected by
+        # _rcv3_check_bulk_delete in order to recover from it.
+        return eff.on(partial(_rcv3_check_bulk_delete, self.lb_node_pairs))
+
+
+_UUID4_REGEX = ("[a-f0-9]{8}-?[a-f0-9]{4}-?4[a-f0-9]{3}"
+                "-?[89ab][a-f0-9]{3}-?[a-f0-9]{12}")
+_RCV3_NODE_NOT_A_MEMBER_PATTERN = re.compile(
+    "Node (?P<node_id>{uuid}) is not a member of Load Balancer Pool "
+    "(?P<lb_id>{uuid})".format(uuid=_UUID4_REGEX),
+    re.IGNORECASE)
+_RCV3_LB_INACTIVE_PATTERN = re.compile(
+    "Load Balancer Pool (?P<lb_id>{uuid}) is not in an ACTIVE state"
+    .format(uuid=_UUID4_REGEX),
+    re.IGNORECASE)
+
+
+def _rcv3_check_bulk_delete(attempted_pairs, result):
+    """Checks if the RCv3 bulk deletion command was successful.
+
+    The request is considered successful if the response code indicated
+    unambiguous success, or the nodes we're trying to remove aren't on the
+    respective load balancers we're trying to remove them from, or if the load
+    balancers we're trying to remove from aren't active.
+
+    If a node wasn't on the load balancer we tried to remove it from, or a
+    load balancer we were supposed to remove things from wasn't active,
+    returns the next step to try and remove the remaining pairs. This is
+    necessary because RCv3 bulk requests are atomic-ish.
```

Comment:
Nitpick - could the params (namely `attempted_pairs`, and order of each tuple) be documented?  It took me a second to figure out why match.groups() was reversed otherwise.  :)